pypeit/par/pypeitpar.py

```diff
@@ -3300,6 +3306,10 @@ def __init__(self, bspline_spacing=None, sky_sigrej=None, global_sky_std=None, n
                                 ' For example: The following string   :10,35:65,80:   would select the' \
                                 'first 10%, the inner 30%, and the final 20% of _all_ slits.'
 
+        defaults['mask_by_boxcar'] = False
+        dtypes['mask_by_boxcar'] = bool
+        descr['mask_by_boxcar'] = 'Mask the sky region around the object by the boxcar radius (set in ExtractionPar).'
```

Comment:
Make it clear this is only for global sky subtraction.

Reply:
edited